inception-project/inception
# #1773 Project import on projects page: add option to add missing users

Allow administrators to also import permissions and create missing users when importing projects on the projects overview page. 

**What's in the PR**
* Re-use of import form from administration page

**How to test manually**
* Admin user should be able to import project from Projects overview with additional checkboxes for users and user permissions
* Normal user should not see the import button at all
* Project Creator should only be able to import the project without users
